srcpkgs/task/template

```diff
@@ -1,17 +1,16 @@
 # Template file for 'task'
 pkgname=task
-version=3.0.2
+version=3.4.1
 revision=1
 build_style=cmake
-build_helper="rust"
```

Comment:
why?

Reply:
removing this broke cross: https://build.voidlinux.org/#/builders/8/builds/3134

Reply:
Was just waiting to see what CI told me since I figured it might have been something wrong locally.

Reply:
Yea I didn't mean to merge it, tried to approve CI and clicked the wrong button...

Reply:
Ah, I see. That makes sense. Thanks for the quick fix. Apologies for such a rocky first contribution---I'll keep this all in mind for future PRs.